hcipy/util/finite_difference.py

```diff
@@ -1,3 +1,5 @@
+from hcipy.field.util import make_uniform_grid
```

Comment:
Merge this with the line below.

Reply:
Done.

Reply:
I'll click resolve after I confirm you made the change. Don't resolve yourself.